mozilla/foundation.mozilla.org
# added animated share buttons

Closes #7985
Related PRs/issues #

**Link to sample test page**: https://foundation-s-animated-s-slhda2.herokuapp.com/en/youtube/regretsreporter/

**Steps to test**: 

1. Visit test page above
2. Click the share this button
3. buttons should animate as expected
4. Clicking on buttons should preform their expected functions with prepopulated copy, and the copy URL button should do so.
5. If they all work, testing is complete!
